api/cedar_metadata_records/views.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import unicode_literals
+import logging
+
+from rest_framework import permissions as drf_permissions
+from rest_framework.exceptions import NotFound
+from rest_framework.generics import ListCreateAPIView, RetrieveUpdateDestroyAPIView
+
+from api.base import permissions as base_permissions
+from api.base.filters import ListFilterMixin
+from api.base.parsers import (
+    JSONAPIMultipleRelationshipsParser,
+    JSONAPIMultipleRelationshipsParserForRegularJSON,
+)
+from api.base.versioning import PrivateVersioning
+from api.base.views import JSONAPIBaseView
+from api.cedar_metadata_records.permissions import CedarMetadataRecordPermission
+from api.cedar_metadata_records.serializers import CedarMetadataRecordsSerializer, CedarMetadataRecordsCreateSerializer
+
+from framework.auth.oauth_scopes import CoreScopes
+
+from osf.models import CedarMetadataRecord
+
+logger = logging.getLogger(__name__)
+
+
+class CedarMetadataRecordList(JSONAPIBaseView, ListCreateAPIView, ListFilterMixin):
+
+    permission_classes = (
+        CedarMetadataRecordPermission,
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+    )
+    required_read_scopes = [CoreScopes.ALWAYS_PUBLIC]
+    required_write_scopes = [CoreScopes.NODE_BASE_WRITE, CoreScopes.NODE_FILE_WRITE, CoreScopes.NODE_REGISTRATIONS_WRITE]
```

Comment:
Come to think about it, if this is in our private namespace, should we even specify the oauth scope for it? Since it shouldn't even support oauth apps.

Reply:
> private endpoint not supporting oauth

I don't think this is a "rule" and remember Oauth makes it easy to bulk add/update things. I'd make highly specific new scopes and put them under "Full read/write" in the scopes module, it will be easier to control the permission later with a specific custom scope. My 2 cents.